WORKSPACE.bazel

```diff
@@ -1,2 +1,32 @@
 # Please notice that any bazel targets and definitions in this repository are currently experimental
 # and for internal use only.
+
+workspace(name = "ql")
```

Comment:
@criemen I would have called this workspace `codeql`, but we used that already... should we do some renaming? Or shall we leave it like this?

Reply:
@criemen as you suggested in chat, I'll rename this workspace to `@codeql` and semmle-code to `@semmle-code`, but I think I'll do it in a separate pair of PRs doing only that. Currently this PR does not depend on the internal one (it's the internal one depending on this), while a bazel workspace rename has it actually the other way around (the codeql repo change would require the `codeql` name to be freed up internally before using it). I would prefer to keep PR dependencies one-way :slightly_smiling_face:

Reply:
right!